ViteMaDose/Networking/APIEndpoint.swift

```diff
@@ -20,17 +20,17 @@ enum APIEndpoint {
 
 extension APIEndpoint: EndpointType {
     var baseURL: URL {
-        return URL(staticString: "https://raw.githubusercontent.com/CovidTrackerFr/vitemadose/data-auto/data/output/")
+        return URL(string: RemoteConfiguration.shared.baseUrl)!
```

Comment:
Tu peux utiliser le custom init qu'on a avec `staticString` pour eviter le force unwrap tu veux 😊 
```suggestion
        return URL(staticString: RemoteConfiguration.shared.baseUrl)
```

Reply:
Non ça ne fonctionne pas, comme c'est pas une `staticString` 😄